functions/helpers/postSurveyJanitor.private.ts

```diff
@@ -0,0 +1,80 @@
+// eslint-disable-next-line prettier/prettier
```

Comment:
It's worth adding some comments to this file also, to be clear about what it's replacing and why that's necessary.